### ISSUE TITLE
Peering Request

### ISSUE THREAD

**Issue by jonaharagon** [routeviews/public-issues#170]
### Full Name

Triplebit

### ASN

401332

### PeeringDB Integration

- [X] My PeeringDB Entries are current
- [ ] Peer at **ALL EXCHANGES** (*and skip the Peer Address(es) field below*)

### Peer Address(es)

204.209.51.53

### Multihop

- [X] Multihop Peering Request

**Comment by pfsinoz** [comment]
Thank you for your interest in RouteViews.

Some more information is needed though... Are you able to send us an IPv6 BGP feed as well? If so, using which IPv6 address?

Also, would you be able to send us the full tables as you see them, or just the single IPv4 /24 and single IPv6 /41 we see in the BGP table right now?

**Comment by jonaharagon** [comment]
Well I could send both address families via IPv4 if your side supports it, or I can send from `2605:8400:ff:13e::1`. I can send the full tables we see.

**Comment by nihb333** [comment]
Hi there.  We prefer a session per protocol. If you agree to send is your full tables, please set up multihop sessions to AS6447: 
        v4: 128.223.51.113
        v6: 2001:468:d01:33::113

We will prepare the sessions on our side shortly
Thanks for the support

**Comment by nihb333** [comment]
requester went silent

**Comment by jonaharagon** [comment]
I'm very sorry, I missed this and forgot to check back. Can we still set this up? I am still interested in supporting the project.

I set up multihop sessions with the details you provided above. If that still works feel free to set up the sessions on your side and let me know. Sorry again and thanks! @nihb333

**Comment by nihb333** [comment]
Hi 
Thanks for picking this back up.  Could you tell us a bit more about your network?  We have changed our peering policy a bit.  According to peeringDB your only IXP is MICE.  Do you have a larger number of peerings there and do you have any private peers?  In order to set up a multihop session, we require that you send us your full view of the Internet. Are you able to do that? 

You can read more about our policy on our blog  [here](https://www.routeviews.org/routeviews/2025/01/10/routeviews-peering-policy/)
Remember that our system is available to you no matter whether you peer or not
BR
Nina

**Comment by jonaharagon** [comment]
Yes I can send a full view. Would you also prefer I use BGP AddPath to send you all paths I know about, or should I only send the best/current path like a normal session?

Part of the reason I wanted to participate is because MICE (or anyone else connected) does not to my knowledge.

Currently I have 119 peers with MICE, 2 private sessions, and 1 downstream customer. I have a session with bgp.tools's route collector so you can see live details here:

https://bgp.tools/as/401332#connectivity

**Comment by nihb333** [comment]
Hi there

Sorry about the delay.  We can set up the peering.  Please do not use BGP addpath, but only send is your current paths.  Please configure your side and let me know when done, then I will bring up our side:

Our information:
asn: 6447
v4: 128.223.51.16/24
v6: 2001:468:d01:33::16/64
fqdn: route-views8.routeviews.org

Nina

**Comment by jonaharagon** [comment]
Hello, the session is up on my end.

asn 401332
v4 204.209.51.53
v6 2605:8400:ff:13e::1

**Comment by nihb333** [comment]
Excellent - we are up and running 
route-views8# sh bg su | include 401332
204.209.51.53   4     401332    142887         7 11654632    0    0 00:04:20       981800        0 Triplebit
2605:8400:ff:13e::1 4     401332     53222         5  9917437    0    0 00:02:59       215298        0 Triplebit
route-views8#
Thanks for the support!

Nina
